src/SportsHub.Business/Services/Implementation/TeamService.cs

```diff
@@ -113,17 +113,13 @@ public async Task<Guid> FindTeamIdBySubcategoryIdAsync(Guid subcategoryId)
             return await _teamRepository.FindTeamIdBySubcategoryIdAsync(subcategoryId);
         }
 
-        public IFormFile ConvertTeamLogo(TeamLogo logo)
+        public TeamLogo ConvertTeamLogo(TeamLogo logo)
```

Comment:
Sorry, but I can barely get what this method is for. Could you explain, please?

Reply:
As I understood from Vitalii's code, it is used to avoid the repetition of code in GetTeamByIdAsync and GetTeamsAsync

Reply:
Still don't get it